scripts/sanitycheck

```diff
@@ -676,6 +676,9 @@ class DeviceHandler(Handler):
 
         if options.west_flash is not None:
             command = ["west", "flash", "--skip-rebuild", "-d", self.outdir]
+            if options.west_runner is not None:
+                command.append("-r")
```

Comment:
prefer "--runner" instead of "-r", self documenting.

Reply:
good point.  fixing.